.orchestra/support/js-prerequisites/npm

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+set -euo pipefail
+
+PORT_FILE=$(mktemp)
+CACHE_FOLDER=$(mktemp -d)
+REPO_PID=0
+SCRIPT_FOLDER="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
+
+cleanup() {
+    trap - SIGINT SIGTERM ERR EXIT
+    if [[ $REPO_PID -gt 0 && -d "/proc/${REPO_PID}" ]]; then
+        kill $REPO_PID
+    fi
+    rm -rf -- "$CACHE_FOLDER"
+}
+trap cleanup SIGINT SIGTERM ERR EXIT
+
+"$SCRIPT_FOLDER/../libexec/static-npm-registry" "$NODE_CACHE" "$PORT_FILE" &
```

Comment:
```
(
  "$SCRIPT_FOLDER/../libexec/static-npm-registry" "$NODE_CACHE" "$PORT_FILE"
  // Do something smart here?
) &
```

Reply:
`static-npm-registry` already handles cleaning up `$PORT_FILE` by itself, there's not much else to do since it's a server